sunkit_pyvista/plotter.py

```diff
@@ -279,7 +288,61 @@ def plot_field_lines(self, field_lines, **kwargs):
             grid = self._coords_to_xyz(field_line.coords.ravel())
             field_line_mesh = pv.StructuredGrid(grid[:, 0], grid[:, 1], grid[:, 2])
             color = {0: 'black', -1: 'tab:blue', 1: 'tab:red'}.get(field_line.polarity)
+            field_line_mesh.add_field_array([color], 'color')
             self.plotter.add_mesh(field_line_mesh, color=color, **kwargs)
             field_line_meshes.append(field_line_mesh)
 
         self._add_mesh_to_dict(block_name='field_lines', mesh=field_line_meshes)
+
+    def save(self, filepath):
+        """
+        Adds all of the meshes in the current dictionary
+        to a :class:`~pyvista.core.MultiBlock`.
+
+        Parameters
+        ----------
+        filepath : `str` or `pathlib.Path`
+            Name of the file to save as, should have vtm or vtmb as an extension.
+
+        Notes
+        -----
+        This saves the rendered plot as a vtm extended file as well as a directory
+        of the individual meshes with the specified name.
+        """
+        file_path = Path(filepath)
+        directory_path = file_path.with_suffix('')
+        if directory_path.exists():
+            raise ValueError(f"Directory '{directory_path.absolute()}' already exists")
```

Comment:
Is it not fine if the directory already exists?

Reply:
It would store the meshes in that directory, suppose the user specified data.vtm and has a directory called `data` in that path, the individual meshes are stored inside the directory, isn't that not what we want?

Reply:
But what if they want to save it in a pre-existing directory?

Reply:
The method saves both the miltiblock as a vtk file and the individual meshes of the same name as that of the vtk file in the specified filepath.
For example, filepath = mapdata.vtm
There's a vtm file named mapdata.vtm and a directory named mapdata with the individual meshes. Now if the user already has a directory named mapdata in the specified filepath, the individual meshes (vtp extension I guess) would be stored in that directory instead

Reply:
Would an argument similar to overwrite where it save in specified directory be useful?

Reply:
Sure.